src/methods.jl

```diff
@@ -14,7 +14,7 @@ const monadic = [deg2rad, rad2deg, transpose, asind, log1p, acsch,
                  trigamma, invdigamma, polygamma, airyai, airyaiprime, airybi,
                  airybiprime, besselj0, besselj1, bessely0, bessely1]
 
-const diadic = [max, min, hypot, atan, mod, rem, copysign,
+const diadic = [max, min, hypot, atan, div, rem, mod, copysign,
```

Comment:
I hope we can directly simplify `div` on construction.

Reply:
How can I do that?

Is the same planned for trig/exp?

Reply:
Check the `src/types.jl` file for Div overloads.